samples/Config.Options.Snapshot.Web/Config.Options.Snapshot.Web.csproj

```diff
@@ -4,7 +4,7 @@
 
   <PropertyGroup>
     <IsPackable>false</IsPackable>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
```

Comment:
Mono is only required if you wanted to _run_ a .NET Framework app on xplat, which we never do. Restore and compile work just fine.